scale/scale/settings.py

```diff
@@ -285,7 +291,7 @@
     'handlers': LOG_HANDLERS,
     'loggers': {
         '': {
-            'handlers': ['console', 'console-err'],
+            'handlers': ['console-err-debug'],
```

Comment:
Why are we forcing all log levels through stderr?

Reply:
@gisjedi and I discussed it, couldn't think of a good reason for logging to show up on more than one output... 
I'm OK changing it back if you think it otherwise

Reply:
In the long distance past we had separate files for error and above and then a full log file. I'm now of the opinion log output is best sent in its entirety to STDERR... STDOUT is more intended for *nix process data piping to downstream STDIN. In any case it's all going to end up in Elasticsearch in the future where it can be uniformly queried and filtered as desired.